pkg/api/util.go

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
```

Comment:
no, some are used in all places (api, core, server). I do not feed quite comfortable with the fragmentation of the API into several files (the Limes pattern). 

I would not object to throw them all into a single api.go file like in Prometheus itself.